gradle/dependencies.gradle

```diff
@@ -13,6 +13,13 @@ repositories {
         name "Rei"
         url "https://maven.shedaniel.me/"
     }
+    maven {
+        name "Mirality"
+        url "https://maven.mirality.co.nz/"
```

Comment:
Dis you now manually upload this to your maven?

Reply:
I did.  And I verified that it works perfectly fine with both the version from cursemaven and the "real" curseforge mod.  It's no different than any other maven.

But if you want to host the files on the LDT maven or elsewhere, that's fine too, I can send them to you or you can pull them yourself.

Reply:
It is not about it working in your case.
It needs to work with our artifact cache

Yes your maven would probably work if it supplied hashing files and etags which I doubt you do. However, you need to manually manage this now undocumented external system which is a no go to start with.

And as @Raycoms discussed there is an official solution from curse with better maven Metadata. So that is what we need you to try.

To be clear: your manual uploaded maven artifact is just as hard a no as the cursemaven solution you had before!

Reply:
I think the official cf maven doesnt have metadata either, but it uses names and stuff

Reply:
Atm the PR doesnt build, but I guess if it builds we should be fine, right?

Reply:
Names is also Metadata it needs to be properly identifyable by our cache and I would love it if they support hash files so that we can properly detect cache invalidation. But proper naming is at least a must. And it had to be automated. We are not doing manual stuff cause then you can just put it in a libs folder and load it from there.

Reply:
It has all the files that ForgeGradle builds for maven:
![image](https://user-images.githubusercontent.com/539951/232473178-5140fa92-ec6c-4564-b72f-ab1088df334c.png)

I agree that an external maven is not ideal, but you already veto'd the curse maven, so idk what you're even asking any more.

Although I just noticed that the build failed in `javadoc`, which I don't habitually run locally.  It looks like the problem there is that the API jar had external links, which is readily solved.

Reply:
Now it did build properly, so it should be fine

Reply:
The build succeeded from cursemaven too 😛

Reply:
Cursemaven just works, and literally everybody that I can find on the entire internet recommends using it over the curseforge.com maven "API", because nobody can figure out how to even access anything using it.  If you can tell me what dependency name it should use, I can change it to that, but what little docs there are on it gives me negative data and the endpoint itself is not explorable, so it's completely worthless.

For my maven, I did actually rebuild the mod (from the matching commit), mostly to get it to generate source packages so that Find Usages works better.  (These *could* work through cursemaven too, except that people don't usually upload them because the Forge MDK doesn't generate them by default.)  But it remains compatible with the "real" mod, so it still works fine.